mkeditorpacks/README.md

```diff
@@ -0,0 +1,19 @@
+MkEditorPacks
+=============
+
+This a tool suite to build game packs for map editors.
+
+It is developped for the [Unvanquished](https://unvanquished.net/) project, but must fit any project supported by the editors listed below.
+
+Supported editors
+-----------------
+
+- [GtkRadiant](http://icculus.org/gtkradiant/)
+- [NetRadiant](https://gitlab.com/xonotic/netradiant)
+
+Authors
```

Comment:
Hi @neumond, what's up? 😃

Let's make it BSD 3-clause, as the script could potentially be used by other Dæmon-based games. @neumond, @illwieckz should we use `Unvanquished Development` (and potentially `Daemon Developers` in case we move this to @DaemonEngine at some point) as the copyright holder or do you prefer our names? Copyrighted years are 2014 and 2016–2017, by the way.